### PR TITLE
Add a few more tests.

### PR DESCRIPTION
**Pull request by statwonk**
Going to add tests by first working my way through functions shown in the readme. I'll add to this today.

EDIT: I see my note re: `id_` failing when given a single id is wrong. Looks like that's `n()`, working in `utils.R` now.

### CODE REVIEW

**Comment by sckott** [comment]
thanks!

**Comment by statwonk** [comment]
Got CI working on Travis. Looks like AppVeyor can't find the ES instance.

**Comment by statwonk** [comment]
Sorry for the nasty commit log! Made a few fumbles with merge. Should I try to squash?

File: R/search.R [hunk 1]
```diff
@@ -12,9 +12,6 @@ Search_ <- function(.obj = "", body = list(), params = list(), ...){
   } else {
     .obj
   }
-  if (as.character(body) == "[]") {
```
Comment (statwonk) [inline R/search.R:15]: @sckott it's unclear to me what this bit tries to achieve. Try `Search_("shakespeare")` to see the error it was throwing.

**Comment by sckott** [comment]
> Should I try to squash?

yes, if you can please

File: R/search.R (hunk 1 above)
Comment (sckott) [inline R/search.R:15]: dont remember - seems like it should be removed, do that if you can or I can do later

**Comment by sckott** [comment]
let me know when commits are squashed

**Comment by statwonk** [comment]
Will do!

**Comment by statwonk** [comment]
@sckott squashed and cleaned up history! There are a few incidental commits prior to "tests." I got Travis passing again, but looks like AppVeyor isn't finding the cluster on `master` (nor here):

```
Error: 
  Failed to connect to http://127.0.0.1:9200
  Remember to start Elasticsearch before connecting
Execution halted
```

**Comment by sckott** [comment]
Great, thanks!  We'll worry about appveyor later, not sure what's wrong there

**Pull request merged**